python/lsst/ts/externalscripts/auxtel/calsys_take_narrowband_data.py

```diff
@@ -23,42 +17,137 @@
 
 __all__ = ["CalSysTakeNarrowbandData"]
 
+
 class CalSysTakeNarrowbandData(scriptqueue.BaseScript):
     """
     """
 
     def __init__(self, index):
         super().__init__(index=index,
                          descr="Configure and take LATISS data using the"
-                               "auxiliary telescope CalSystem.",
-                         remotes_dict={'electrometer': salobj.Remote(SALPY_Electrometer, 1),
-                                       'monochromator': salobj.Remote(SALPY_ATMonochromator),
-                                       'fiber_spectrograph': salobj.Remote(SALPY_FiberSpectrograph),
-                                       'atcamera': salobj.Remote(SALPY_ATCamera),
-                                       'atspectrograph': salobj.Remote(SALPY_ATSpectrograph),
-                                       'atarchiver': salobj.Remote(SALPY_ATArchiver)
-                                      })
+                               "auxiliary telescope CalSystem.")
         self.cmd_timeout = 60
         self.change_grating_time = 60
+        self.electrometer = salobj.Remote(domain=self.domain, name="Electrometer")
```

Comment:
I think you need `index=1` for the electrometer,  based on reading the original code.